src/main/java/org/mockito/internal/util/Platform.java

```diff
@@ -40,6 +47,18 @@ public static String describe() {
                              JVM_INFO,
                              OS_NAME,
                              OS_VERSION);
+        if (isAndroid()) {
+            description = join(
+                    "IMPORTANT INFORMATION FOR ANDROID USERS:",
```

Comment:
Fantastic message!

It would be great to link to official documentation, for example: https://javadoc.io/page/org.mockito/mockito-core/latest/org/mockito/Mockito.html#0.1